solr/solrj/src/java/org/apache/solr/common/util/ExecutorUtil.java

```diff
@@ -314,15 +314,19 @@ public void execute(final Runnable command) {
               if (t instanceof OutOfMemoryError) {
                 throw t;
               }
-              if (enableSubmitterStackTrace) {
```

Comment:
Don't we still need to have this flag?

Reply:
I don't believe so. So the `baseCause`, which is now always logged, will be `t` (the real throwable) if `submitterStackTrace` is `null`. Since when `enableSumbitterStackTrace==false` then `submitterStackTrace=null` (logic from above), then we will only be logging `t` if `enableSubmitterStackTrace==false`. The submitterStackTraces will only be included if `enableSubmitterStackTrace==true`.

It is still a little different than the previous logic, which only logged the error message, but I think the spirit of the flag is still the same.